platform/security/encryption/platform-security-encryption-impl/src/main/java/ddf/security/encryption/impl/EncryptionServiceImpl.java

```diff
@@ -94,62 +105,60 @@ public synchronized String decrypt(String encryptedValue) {
    * }</pre>
    */
   // @formatter:on
+  @Override
   public String decryptValue(String wrappedEncryptedValue) {
-    String decryptedValue = null;
     String encryptedValue = unwrapEncryptedValue(wrappedEncryptedValue);
-
     if (wrappedEncryptedValue == null) {
       LOGGER.debug("A null password was provided.");
-      decryptedValue = null;
-    } else if (wrappedEncryptedValue.isEmpty()) {
+      return null;
+    }
+    if (wrappedEncryptedValue.isEmpty()) {
       LOGGER.debug("A blank password was provided in the configuration.");
-      decryptedValue = "";
-    } else if (!wrappedEncryptedValue.equals(encryptedValue)) {
-      LOGGER.debug("Unwrapped encrypted password is now being decrypted");
-      decryptedValue = decrypt(encryptedValue);
-    } else if (wrappedEncryptedValue.equals(encryptedValue)) {
-      /**
-       * If the password is not in the form ENC(my-encrypted-password), we assume the password is
-       * not encrypted.
-       */
-      decryptedValue = wrappedEncryptedValue;
+      return "";
+    }
+    // If the password is not in the form ENC(my-encrypted-password),
+    // we assume the password is not encrypted.
+    if (wrappedEncryptedValue.equals(encryptedValue)) {
       LOGGER.warn(
-          "A plain text password was provided in the configuration in the console.  Consider using an encrypted password.");
+          "A plain text password was provided in the configuration in the console. "
+              + "Consider using an encrypted password.");
```

Comment:
We need to revisit this error message. Since encryption happens automatically now, is it still necessary?